workers/generic-worker/runtime_multiuser.go

```diff
@@ -0,0 +1,12 @@
+//go:build multiuser
+
+package main
+
+import (
+	"github.com/taskcluster/taskcluster/v57/workers/generic-worker/process"
+	"github.com/taskcluster/taskcluster/v57/workers/generic-worker/runtime"
+)
+
+func gwVersion() (*process.Command, error) {
```

Comment:
It would be good to add a code comment that we do this as the task user, because really this is just checking that gw binary is executable, rather than just being a helper to see what the generic-worker version is (that we know already from the `version` global variable in the main package, iirc.